hadoop-hdfs-project/hadoop-hdfs-native-client/src/main/native/libhdfspp/lib/common/cancelable.h

```diff
@@ -15,32 +15,27 @@
  * See the License for the specific language governing permissions and
```

Comment:
I have no idea why github thinks this is a rename of inputstream.cc to cancelable.h.  This is a new file.